modules/core/arrow-data/src/test/kotlin/arrow/instances/ComposedInstancesTest.kt

```diff
@@ -34,17 +35,34 @@ class ComposedInstancesTest : UnitSpec() {
           })
       }
 
+    val EQ_OPTION_FN1: Eq<NestedType<ForOption, Conested<ForFunction1, Int>, Int>> = Eq { a, b ->
+      a.unnest().fix() == b.unnest().fix()
+    }
+
+    val EQ_TUPLE2: Eq<Kind2<Nested<ForTuple2, ForTuple2>, Int, Int>> = Eq { a, b ->
+      a.biunnest().fix() == b.biunnest().fix()
+    }
+
     val cf: (Int) -> Kind<Nested<ForOption, ForNonEmptyList>, Int> = { Some(it.nel()).nest() }
 
+    val cf2: (Int) -> Kind<Nested<ForOption, Conested<ForFunction1, Int>>, Int> = { x: Int ->
+      Some({ y: Int -> x + y }.k().conest()).nest()
+    }
+
+    val bifunctorCf: (Int) -> Kind2<Nested<ForTuple2, ForTuple2>, Int, Int> = { Tuple2(Tuple2(it, it), Tuple2(it, it)).binest() }
 
     testLaws(
+      InvariantLaws.laws(ComposedInvariant(Option.functor(), NonEmptyList.functor()), cf, EQ_OPTION_NEL),
+      InvariantLaws.laws(ComposedInvariantCovariant(Option.functor(), NonEmptyList.functor()), cf, EQ_OPTION_NEL),
+      InvariantLaws.laws(ComposedInvariantContravariant(Option.functor(), Function1.contravariant<Int>()), cf2, EQ_OPTION_FN1),
```

Comment:
TestLaws deduplicates tests based on name. That means only one of these will be run. Split 2 of them into a new call to testLaws.